client/llb/fileop.go

```diff
@@ -336,6 +342,35 @@ func Mkfile(p string, m os.FileMode, dt []byte, opts ...MkfileOption) *FileActio
 	}
 }
 
+type SymlinkInfo struct{}
+
+type SymlinkOption interface {
+	SetSymlinkOption(*SymlinkInfo)
+}
+
+func Symlink(target, linkpath string) *FileAction {
```

Comment:
Looks like the function needs to be modified to accept `...SymlinkOption`.

Reply:
done